clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/UpsertAmazonLoadBalancerClassicAtomicOperationSpec.groovy

```diff
@@ -557,4 +561,35 @@ class UpsertAmazonLoadBalancerClassicAtomicOperationSpec extends Specification {
 
     1 * loadBalancing.createLoadBalancer(_ as CreateLoadBalancerRequest) >> new CreateLoadBalancerResult(dNSName: 'dnsName1')
   }
+
+  void "should enable AWS Shield protection if external ELB"() {
```

Comment:
Possibly weak test in that you're only testing the `external` case. 

Generally not a huge fan of all the unrelated mocks, you could avoid if you refactored out the shield bits into a `enableShield(description)`. Only downside with that is you wouldn't get coverage that `enableShield()` was actually invoked.

Your call ... otherwise LGTM.